### PR TITLE
[#6] As a user, I can have the application support multiple locales

### PR DESCRIPTION
**Pull request by luongvo**
https://github.com/nimblehq/flutter_templates/issues/6

## What happened 👀

Configure localization, as a default, we should support `en/th/vi` - depending on the real app development, the developer can add more locale if they want.
 
## Insight 📝

- Follow the guide from https://flutter.dev/docs/development/accessibility-and-localization/internationalization to setup i18n.
- Add support to have `en`, `th` and `vi` translations. 
 
## Proof Of Work 📹

- EN

- VI

- TH

### CODE REVIEW

File: pubspec.lock [hunk 1]
```diff
@@ -209,6 +209,11 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "2.0.0"
+  flutter_localizations:
+    dependency: "direct main"
+    description: flutter
+    source: sdk
+    version: "0.0.0"
```
Comment (doannimble) [inline pubspec.lock:216]: @luongvo I think we only need two line for setup flutter_localizations as : 
flutter_localizations:
    sdk: flutter

File: pubspec.lock [hunk 2]
```diff
@@ -261,6 +266,13 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "4.0.0"
+  intl:
+    dependency: "direct main"
+    description:
+      name: intl
+      url: "https://pub.dartlang.org"
+    source: hosted
+    version: "0.17.0"
```
Comment (doannimble) [inline pubspec.lock:275]: @luongvo Same above, do we only need to configure as : `intl: ^0.17.0` ?

Comment (sleepylee) [reply pubspec.lock:275]: pub.lock is auto generated with the updated dep from pubspec.yaml @doannimble 😂

Comment (doannimble) [reply pubspec.lock:275]: @sleepylee ahhh, then it should be okay 😆

**Review by doannimble — APPROVED** [review]
LGTM

**Review by manh-t — APPROVED** [review]
LGTM 🚀

**Comment by sleepylee** [comment]
last man rebase 🙋

**Comment by luongvo** [comment]
> last man rebase 🙋

easy, done @sleepylee

**Review by sleepylee — APPROVED** [review]

**Pull request merged**